akka-cluster/src/main/scala/akka/cluster/Member.scala

```diff
@@ -32,7 +33,11 @@ class Member private[cluster] (
     case m: Member ⇒ uniqueAddress == m.uniqueAddress
     case _         ⇒ false
   }
-  override def toString = s"Member(address = ${address}, status = ${status})"
+  override def toString =
+    if (team == ClusterSettings.DefaultTeam)
+      s"Member(address = $address, status = $status)"
+    else
+      s"Member(address = $address, team = $team, status = $status)"
 
   def hasRole(role: String): Boolean = roles.contains(role)
 
```

Comment:
yeah, I think team must be passed in to these functions
I don't know why these are public :(